examples/nidcpower/measure-record.py

```diff
@@ -93,7 +77,8 @@ def check_for_initialization_error(response):
         )
     )
     vi = initialize_with_channels_response.vi
-    check_for_initialization_error(initialize_with_channels_response)
+    if initialize_with_channels_response.status > 0:
```

Comment:
Note: Since initialization no longer needs to check for errors, just warnings, it felt cleaner to just inline the check. I could keep this as a separate `check_for_warning` helper if desired. I'm not sure how much it might be used elsewhere in the other examples though.

Reply:
Warnings have error messages too. You can use `client.ErrorMessage()` to query what they are.